Horan_Colby/Assignments/survey/templates/index.html

```diff
@@ -0,0 +1,33 @@
+<html>
+    <head>
+       <title></title>
+    </head>
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='index.css') }}">
+
+    <body>
+        <h1>Dojo Survey</h1>
+        <form action='/users' method='post'>
+            Name: <input id='name' type='text' name='name'>
+            Favorite Language: <select id ='favelang' name='favorite_language'>
+            <option value="{{ 'PYTHON' }}">{{ 'PYTHON' }}</option>
+            <option value="{{ 'RUBY' }}">{{ 'RUBY' }}</option>
+            <option value="{{ 'JAVASCRIPT' }}">{{ 'JAVASCRIPT' }}</option>
+            <option value="{{ 'PHP' }}">{{ 'PHP' }}</option></select>
```

Comment:
what the 😑?

{{ 'PYTHON' }} is identical to just typing PYTHON; since this is a constant (e.g. will never change), it's better not to use {{}} for it and just type it as regular text. Unless, am I missing something here--? What were you trying to do ?_____?